frontends/pytorch/csrc/builder/ivalue_importer.cpp

```diff
@@ -248,6 +248,17 @@ MlirValue IValueImporter::rawImportIValue(c10::IValue ivalue) {
                                  npcompListTypeGet(context), elems);
     return mlirOperationGetResult(operation, 0);
   }
+  if (ivalue.isTuple()) {
```

Comment:
this change should be tested analogously to frontends/pytorch/test/module_import/list.py (create module_import/tuple.py)

Reply:
SGTM, I added the corresponding list/tuple test(s) in favor of more downstream coverage from the incoming integration test_suite (i.e. model curriculum)